client/src/index.css

```diff
@@ -323,6 +323,108 @@ footer .icon i {
   margin: 20px auto;
 }
 
+.controls {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 1rem;
```

Comment:
Add flex-wrap and gap here to make it resonsive

Reply:
Done.

![image](https://github.com/yodkwtf/cinematica-mern/assets/109443065/7c073820-ec3b-4add-a232-02169d5174e0)